site/content/docs/5.2/migration.md

```diff
@@ -134,14 +132,14 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
   </details>
 
 - <details class="mb-2">
-    <summary><span class="badge bg-danger">Breaking</span> Deprecated CSS variables:</summary>
+    <summary><span class="badge text-danger-emphasis bg-danger">Breaking</span> Deprecated CSS variables:</summary>
```

Comment:
Should we introduce this `.text-danger-emphasis` on all the badges ? Might be done on Bootstrap side first IMO.

Reply:
Tried it in https://github.com/twbs/bootstrap/pull/38013. In the meantime, let's keep these badges close to Bootstrap in this PR.